pulsar-ai-tools/README.md

```diff
@@ -0,0 +1,63 @@
+
+Parameters:
+
+| Name             | Description                                                                                                         |
+|------------------|---------------------------------------------------------------------------------------------------------------------|
+| model            | The model to use, it depends on the provider                                                                        |
+| compute-service  | The embeddings provider (openai, huggingface...)                                                                    | 
+| text             | Template for the text to compute the embeddings on. You can use the Mustache syntax (for example {{value.field1}} ) |
+| embeddings-field | The name of the field to add or update (for example value.embeddingsvector)                                         |
+
+
+### Query
+
+Step name: `query`
+
+Executes a query against the database configured in the "datasource" section.
+
+Parameters:
+
+| Name         | Description                                                                 |
+|--------------|-----------------------------------------------------------------------------|
+| output-field | The name of the field to add or update (for example value.embeddingsvector) |
+| query        | Template for the query, you can use '?' for parameters                      |
+| fields       | An array of field names to use to fill-in the parameters of the query       |
+
+
+### AI Chat Completion
+
+Step name: `ai-chat-completions`
+
+Calls a Completion AI model to generate a text completion.
+
+Parameters:
+
+| Name              | Description                                                                                                                                                                                                                                 |
+|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| output-field      | The name of the field to add or update (for example value.embeddingsvector)                                                                                                                                                                 |
+| messages          | Template for the messages, it is an array. You can use Mustache in order to fill in the template with the fields of the message.                                                                                                            |
+| model             | ID of the model to use. See the [model endpoint compatibility](https://platform.openai.com/docs/models/model-endpoint-compatibility) table for details on which models work with the Chat API.                                              |
```

Comment:
Models have different names on Azure. We could also put a link for Azure ?

Reply:
good idea